src/controllers/rows.js

```diff
@@ -1,11 +1,11 @@
 const db = require('../db/index');
```

Comment:
@AbegaM - there are lots of cosmetic (trailing comma) changes in this file, which make it harder to review. Plus, the next person's PR will probably try to fix them - more unnecessary changes.

We need to understand why some blocks of working code have been commented out or changes made.  No problem if this is still WIP though.

Reply:
There is some inconsistency in `Soul` for example, some files are using `double quotes` and other files are using `single quotes` and there are some trailing commas in some files but not in the others, and i have configured my formatter to remove trailing commas and that's why the changes are here, but regarding the lines i commented i can remove the commented code